recipes/street_easy/build_rental_sales_index.py

```diff
@@ -12,6 +13,7 @@
     url_sales_index, dtype=str, compression="zip", index_col="Month"
 )
 
+# Unpivotting tables
```

Comment:
@mgraber  I think this is bad and it would break everything!
I think this would be a better approach:
```python
print('hello')
```